test/unit/mdc-tab-indicator/sliding-foundation.test.js

```diff
@@ -56,7 +56,9 @@ test('#activate does not modify transform and does not transition if no client r
 });
 
 test(`#deactivate removes the ${MDCSlidingTabIndicatorFoundation.cssClasses.ACTIVE} class`, () => {
-  const {foundation, mockAdapter} = setupTest();
+  const foundation = new MDCSlidingTabIndicatorFoundation();
```

Comment:
Why was this test changed to use a real adapter with one mocked method?

Reply:
Code coverage